Workshop_2/Leksjon_Bonus/README.md

```diff
@@ -1,12 +1,84 @@
 # Bonus leksjon
 Her en noen ekstra-oppgaver dersom du skulle bli raskt ferdig.
 
-## Flere miljøer med infrastruktur
-
-Sett opp release-pipelines for qa og produksjon i infrastruktur-prosjektet. 
+## Tags 
+Forsøk å tagge alle ressurser i med App = Workshop2. 
+Etter at det er blitt deployet, kan du gå i portalen og forsøke å finne alle ressurser som du har gitt med den taggen.
 
 ## Slot-deployment i produksjon
 
 Slots er en feature i Web Apps som gjør at du kan deploye til en annen slot for å teste den nye versjonen din, for så å swappe denne inn når du har fått testet denne.
 
 Se om du kan få til å sette opp dette i produksjon. Da må du både oppdatere ARM-templates slik at en slot oppdateres, og i tillegg oppdatere deployment-steget av koden i produksjon til å deploye til en slot i stedet for hoved-websiten.
+
+## Azure Rbac For tilgang til Keyvault for Managed identity. 
```

Comment:
Jeg tenker dette tematisk hører hjemme i WS3 (sikkerhet). Og siden vi har ødelagt for leksjon 1 i ws 3 ved å legge på SAS-tokens allerede i WS1 tenker jeg denne kan være en bra erstatning for WS1 (Connection string til managed identity)

Reply:
Jepp, jeg skal ta og fjerne det :)